spyder/widgets/reporterror.py

```diff
@@ -246,8 +249,18 @@ def __init__(self, parent=None, is_report=False):
         layout.addWidget(self.details)
         layout.addWidget(self.desc_chars_label)
         layout.addSpacing(15)
-        layout.addWidget(self.dismiss_box)
-        layout.addSpacing(15)
+
+        if not self.is_report:
+            layout.addWidget(self.dismiss_box)
+            layout.addSpacing(15)
```

Comment:
```suggestion
```

There's no need of this because it introduces unnecessary spacing between the checkboxes.